tests/common/dependency_helper.h

```diff
@@ -0,0 +1,82 @@
+#pragma once
+
+#include <optional>
+
+#include <winrt/base.h>
+#include <winrt/Windows.Management.Deployment.h>
+#include <winrt/Windows.ApplicationModel.h>
+#include <winrt/Windows.Foundation.Collections.h>
+
+// Helper for querying installed package. Should be called from packge which has
+// restricted capability "packageQuery" in their Manifest
+
+using namespace winrt::Windows::Management::Deployment;
+using namespace winrt::Windows::ApplicationModel;
+using namespace winrt::Windows::Foundation::Collections;
+
+constexpr const wchar_t* VCLibsName = L"Microsoft.VCLibs.140.00_8wekyb3d8bbwe";
+
+struct PackageModel
+{
+    std::wstring pkg_name;
+    std::wstring pkg_version;
+    std::wstring pkg_publisher;
+    std::wstring pkg_install_path;
+};
+
+inline std::optional<PackageModel> query_package_with_name(std::wstring_view pkgFamilyName)

```

Comment:
Do not use `_` in name.  `QueryPackageWithName`

Reply:
Updated